marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/default/lib.sjs

```diff
@@ -15,7 +15,12 @@ function getModel(targetEntity, version = '0.0.1') {
 }
 
 function getMapping(mappingName) {
-  return fn.head(cts.search(cts.andQuery([cts.collectionQuery('http://marklogic.com/data-hub/mappings'), cts.jsonPropertyValueQuery('name', mappingName, ['case-insensitive'])]), ["unfiltered", cts.indexOrder(cts.uriReference(), "descending")]));
+  //Look in new collection, if artifact is not present fallback to the old collection
+  let mapping =  fn.head(cts.search(cts.andQuery([cts.collectionQuery('http://marklogic.com/data-hub/mapping-artifact'), cts.jsonPropertyValueQuery('name', mappingName, ['case-insensitive'])]), ["unfiltered", cts.indexOrder(cts.uriReference(), "descending")]));
```

Comment:
Assuming we do need a new collection (that's a separate question), should at least avoid the duplication of the query construction and make a function that accepts a collection name and returns the cts.andQuery.